### ISSUE TITLE
Add Highway env

### ISSUE THREAD

**Issue by rbange** [JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl#89]
https://github.com/eleurent/highway-env looks like very interesting environments. The Maintainer also released the "[first stable release](https://github.com/eleurent/highway-env/releases/tag/v1.0)".

Implementation should not be too hard as it uses the `gym` api?

**Comment by findmyway** [comment]
Yes I think it should be supported by `GymEnv` (through PyCall) out-of-box. But I'll verify it when I have time.

By the way, you may also be interested in https://github.com/sisl/AutomotiveDrivingModels.jl and https://github.com/sisl/AutoViz.jl
